test/e2e/downwardapi_volume.go

```diff
@@ -81,11 +83,30 @@ var _ = Describe("Downward API volume", func() {
 		},
 			podLogTimeout, poll).Should(ContainSubstring("key1=\"value1\"\n"))
 
-		//modify labels
-		pod.Labels["key3"] = "value3"
-		_, err = f.Client.Pods(f.Namespace.Name).Update(pod)
-		Expect(err).NotTo(HaveOccurred())
-
+		//modify labels	in a update/get loop
```

Comment:
I think in general it should be okay to force the update for this test -- there should be no other actor material to the test updating the pod.